bitrise.yml

```diff
@@ -24,9 +24,16 @@ workflows:
         - branch: master
     - path::./:
         title: Step Test - iOS archive
+        is_always_run: true
```

Comment:
Why is this needed?

Reply:
This is to make sure the next step is run. currently, the test step fails because of provisioning profile issues on my machine. works fine on the ci